assets/src/stories-editor/helpers/index.js

```diff
@@ -144,9 +144,6 @@ const getDefaultMinimumBlockHeight = ( name ) => {
 		case 'core/embed':
 			return 200;
 
-		case 'core/image':
```

Comment:
Removed this since it looks like `getDefaultMinimumBlockHeight` is not called for image blocks anyway.

Reply:
@miina How does that change to `assets/src/block-editor/helpers/index.js` affect the stories editor?